src/content/docs/pt-br/homes/platform-home.mdx

```diff
@@ -0,0 +1,49 @@
+---
+type: homepage
+permalink: /documentacao/produtos/gestao-de-contas/
```

Comment:
será que deveria ser só Contas? e pedirmos redirects? tentando pensar se é tudo gestão

Reply:
Sabe que eu to com essa dúvida também, e sobre algumas outras páginas. Eu acho que a URL e os títulos não estão batendo, queria sentar com Juliano e conferir isso.